tests/ImageTests.php

```diff
@@ -447,7 +460,12 @@ protected function output($file)
     public function setUp()
     {
         $dir = $this->output('');
-        `rm -rf $dir`;
+        
+        if (PHP_OS === 'WINNT') {
+            `rd /s /q "$dir"`;
```

Comment:
Because rmdir is not recursive
But I agree this is not really good to have extern commands like this